automated/linux/secure-boot/secure-boot-enable.yaml

```diff
@@ -0,0 +1,11 @@
+metadata:
+  format: Lava-Test Test Definition 1.0
+  name: secure-boot
+  description: "Check SecureBoot is enabled"
+  os:
+    - linux-any
+  scope:
+    - functional
+run:
+  steps:
+    - lava-test-case "secure-boot-enabled" --shell test "$(efivar -d -n 8be4df61-93ca-11d2-aa0d-00e098032b8c-SecureBoot)" -eq 1
```

Comment:
Could you convert this to a script that takes a parameter so it's reusable?

Reply:
@mwasilew I created the script without the parameter because the variable is hard coded. I have put more detail in the description